.bash_nav

```diff
@@ -7,3 +7,33 @@ alias desk='cd /mnt/c/Users/gsera/Desktop'
 # source bash
 
 alias srcbash='source ~/.bashrc'
+
+# ---This alias effectively creates a bookmarking app---
+# it adds global symlink shortcuts for use with the 'cds' command
+# ex: 'cds studycloud' will run 'cd -P ~/.short/studycloud'
+# to add a shortcut: go to the target directory and run 'cds + <shortcut_name>' or 'cds + .' to use the current directory name
+# to remove a shortcut: run 'cds - <shortcut_name>' (you can also tack on multiple shortcut names separated by spaces)
+# to list your shortcuts: run 'cds .'
+cds() {
+    if [ ! -d ~/.short ]; then
+        mkdir ~/.short
+    fi
+    if [ $# -eq 1 ] && [ $1 = "." ]; then
+        ls ~/.short/
+    else
+        if [ $# -eq 2 ] && [ $1 = "+" ]; then
+            ln -s "$PWD" "$HOME/.short/$2"
+        elif [ $# -ge 2 ] && [ $1 = "-" ]; then
+            local args=( ${@:2} )
+            eval "rm ${args[@]/#/"~/.short/"}"
```

Comment:
Here, we break up each element in the `args` array and prepend `~/.short/` to each one. Spaces are also automatically added in between. The final result is a string `rm ~/.short/shortcut_name1 ~/.short/shortcut_name2` etc. We call `eval` on this string to evaluate it as code.